### DISCUSSION TITLE
Support a `flux get all` and/or `flux get <command> all` command

### DISCUSSION THREAD

**Discussion by darksinge** [fluxcd/flux2#1114] [Proposals]
This is a feature request to add a `flux get all` command to the CLI.

This would be a nice quality-of-life enhancement to the CLI. Sometimes I want to quickly peak at the status of the various toolkit components. I also figure if `kubectl get all` has a use case in the Kubernetes world, then `flux get all` seems like a natural fit.

For example, the following is a bit cumbersome:
```bash
$ flux get kustomizations
$ flux get image update
$ flux get image policy
$ flux get image repository
$ flux get sources bucket
$ flux get sources git
$ flux get sources chart
...etc
```

As opposed to something more ergonomic such as the following:
```bash
$ flux get all
NAME          			READY	MESSAGE                                                          	REVISION                                       	SUSPENDED 
kustomize/apps          	True 	Applied revision: master/1cca45c7d735483ed7f40d11750048d3c159c7f3	master/1cca45c7d735483ed7f40d11750048d3c159c7f3	False    	
kustomize/flux-system   	True 	Applied revision: master/1cca45c7d735483ed7f40d11750048d3c159c7f3	master/1cca45c7d735483ed7f40d11750048d3c159c7f3	False    	
kustomize/infrastructure	True 	Applied revision: master/1cca45c7d735483ed7f40d11750048d3c159c7f3	master/1cca45c7d735483ed7f40d11750048d3c159c7f3	False    	

NAME       				READY	MESSAGE                                                          	REVISION                                       	SUSPENDED 
source.gitrepositories/flux-system	True 	Fetched revision: master/1cca45c7d735483ed7f40d11750048d3c159c7f3	master/1cca45c7d735483ed7f40d11750048d3c159c7f3	False    	

NAME  			READY	MESSAGE                                                   	REVISION                                	SUSPENDED 
source.helm/stable	True 	Fetched revision: 1e23f3716d882179659d5485376a28f87b0b686a	1e23f3716d882179659d5485376a28f87b0b686a	False    	

...
```

Something like `flux get images all` and `flux get sources all` would be welcome additions too. It probably goes without saying, but this wouldn't apply to commands that don't have multiple components under a namespace. For instance, `flux get ks all` would be superfluous since `flux get ks` already gives you all `Kustomization`s.

---

And I guess for now I'll just settle for `$ alias fluxgetall="flux get ks && flux get source git && flux get ..."` :wink:

**Comment by stefanprodan** [comment]
The `flux get sources all` and `flux get images all` commands have been implemented in https://github.com/fluxcd/flux2/pull/1116 and will be available in flux2 v0.10.
